### ISSUE TITLE
How to plot accuracy or loss for training and validation data

### ISSUE THREAD

**Issue by nadamaguid** [cerndb/dist-keras#64]
It is unclear how a graph plot can be generated showing the accuracy or loss over epochs. With keras, the history dictionary maintains the values across epochs. After drilling through I figured out that there is a history list but it is unclear what the values being maintained represent. Your help would be much appreciated

**Comment by Roamoin** [comment]
answered in - [#33](https://github.com/cerndb/dist-keras/issues/33)
I also want to know the loss curve of the validation data and prevent overfit
